client/my-sites/plans/jetpack-plans/product-lightbox/style.scss

```diff
@@ -4,11 +4,19 @@
 	width: 85%;
 	height: auto;
 	background-color: #fff;
-	transition: all 0.2s ease-out;
+	transition: all 0.35s ease-out;
```

Comment:
@elliottprogrammer My mistake I should have written the details for both :) 

For the overlay we want `.35s ease-out`
For the container (that has translateY) we'll want .`25s ease`

Thanks again!

Reply:
Ahh excellent! I was wondering about that.  Ok great, np problem! Thanks!
Done. 🙂